.github/workflows/ci.yml

```diff
@@ -1,106 +1,105 @@
 name: CI
 
-on:
-  push:
-    branches:
-      - master
-  pull_request:
-    branches:
-      - master
+on: [push, pull_request]
```

Comment:
@MSP-Greg we changed this intentionally to `master` because having it set this way doubles up the number of checks on a build.

Reply:
Any idea why?  It's used elsewhere.

Without it, one can't run CI in forks, which I did to test it before the PR...

Reply:
When you are pushing branches directly to the main repo on a PR, it shows checks for both `push` and for `pull_request`. I'm not sure if there's a better way to configure it, but it was creating a lot of noise, especially with the already flaky builds.

Reply:
A quick look, and I don't see any hooks, when this is okay maybe try merging and see what happens?

Reply:
You don't see them because you're pushing from a fork. When you have commit bit and push directly to branches on the main repo they show up.

Reply:
Well, if your sure about that, I can revert.

Reply:
Yeah it would be good to remove this part of the change. I made the change because [I was seeing nearly 70 checks](https://github.com/rmagick/rmagick/pull/1005). I switched it to this after seeing what [some other repos do](https://github.com/microsoft/TypeScript/blob/888cd4db20bd34a18b51f7c6a9a216a9a361afea/.github/workflows/ci.yml#L3-L11). You can always open a PR to see the build if you need to.

Reply:
I'm missing something as I'm not seeing the 70 in the link.

But, things get weird sometimes with workflow pushes.  Note that this PR excludes a job, and it does not appear in the main list.  But, it's added at the bottom with a 'required' tag.  also, if one renames jobs, they can double up...

Anyway, I reverted.

Reply:
The reason there's not 70 on that one is because it included the change. I ended up rebasing a bunch of other branches after that was merged, since they were so flaky.